src/Components/Lifecycle/lifecycle.tsx

```diff
@@ -146,7 +121,7 @@ const LifecycleTab: React.FC<React.PropsWithChildren> = () => {
 
   // const emptyAppLifecycleChanges: dataAppLifeCycleChanges[] = []
   // const [AppLifecycleChanges, setAppLifecycleChanges] = useState(emptyAppLifecycleChanges);
-  const [AppLifecycleChanges, setAppLifecycleChanges] = useState<dataAppLifeCycleChanges | null>(null);
+  const [AppLifecycleChanges, setAppLifecycleChanges] = useState<AppLifecycleChanges[] | null>(null);
```

Comment:
Changed this to type that is array type rather than { data: array[] } type since objects are not iterable.